src/CognitiveServices/CognitiveServices/help/Add-AzCognitiveServicesAccountNetworkRule.md

```diff
@@ -0,0 +1,226 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CognitiveServices.dll-Help.xml
+Module Name: Az.CognitiveServices
+online version:
+schema: 2.0.0
+---
+
+# Add-AzCognitiveServicesAccountNetworkRule
+
+## SYNOPSIS
+Add IpRules or VirtualNetworkRules to the NetworkRule property of a Cognitive Services account
+
+## SYNTAX
+
+### NetWorkRuleString (Default)
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String>
+ -VirtualNetworkResourceId <String[]> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### IpRuleObject
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String> -IpRule <PSIpRule[]>
+ [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### NetworkRuleObject
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String>
+ -VirtualNetworkRule <PSVirtualNetworkRule[]> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### IpRuleString
+```
+Add-AzCognitiveServicesAccountNetworkRule [-ResourceGroupName] <String> [-Name] <String>
+ -IpAddressOrRange <String[]> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
```

Comment:
@yangyuan please add a description in the new markdown files

Reply:
@yangyuan this comment was not addressed

Reply:
@yangyuan ignore this -- there was a weird caching problem with GitHub that caused the old changes to be shown. This has been resolved.